doap_Traffic_Control.rdf

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl"?>
+<rdf:RDF xml:lang="en"
+	xmlns="http://usefulinc.com/ns/doap#" 
+	xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" 
+	xmlns:asfext="http://projects.apache.org/ns/asfext#"
+	xmlns:foaf="http://xmlns.com/foaf/0.1/">
+<!--
+		Licensed to the Apache Software Foundation (ASF) under one or more
+		contributor license agreements.  See the NOTICE file distributed with
+		this work for additional information regarding copyright ownership.
+		The ASF licenses this file to You under the Apache License, Version 2.0
+		(the "License"); you may not use this file except in compliance with
+		the License.  You may obtain a copy of the License at
+	 
+				 https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
For legal reasons, [the source file header](https://www.apache.org/legal/src-headers.html#headers) still includes a HTTP link, not HTTPS. They're trying to change it, though. From [LEGAL-536](https://issues.apache.org/jira/browse/LEGAL-536):

>I agree that the text should be changed on the license (several places on the Apache site and internal documents).
>
>I also believe that a license with either the http:// or the https:// should be treated the same.
>
>But we need to be sensitive to many external entities that compare third parties' licenses with the Apache license and a small change like this is likely to cause confusion.
>
>First we need a plan to notify third parties about the change that we plan to make so they are not blindsided. 

LEGAL-536 was last updated 4 days ago, so they're actively working on this. Mailing list thread: https://lists.apache.org/thread.html/r2476215532a0ac81d7875f1e1d69c2b36e492348265ce481a5c99a6c%40%3Clegal-discuss.apache.org%3E

Reply:
I'm sure it's fine. That header was generated for me by Apache's own [DOAP creation tool](https://projects.apache.org/create.html).

Reply:
Okay